.github/CHANGELOG.md

```diff
@@ -166,6 +166,9 @@ This is now fixed.
 a pure dm was converted to ket).
 [(#184)](https://github.com/XanaduAI/MrMustard/pull/184)
 
+* The ray related tests sometimes hangs in github action. This is an attempt to address it.
```

Comment:
I'd change the wording here to adhere more to the CHANGELOG style. I always assume people is reading this in the future so I write it in past tense.
```suggestion
* The ray related tests were hanging in github action causing test to halt and fail. Now ray is forced to init with 1 cpu when running tests preventing the issue.
```